server/sonar-web/src/main/webapp/WEB-INF/lib/need_authentication.rb

```diff
@@ -76,7 +76,7 @@ def external_auth(username, password, servlet_request, details)
     if @java_authenticator
```

Comment:
Please ignore the changes in this files. These changes are done as part of the SONAR-6856 ( We have created separate pull request for this)

SONAR-6856: Add a new API ExternalGroupsProvider.doGetGroups(Context context) to allow plugins to pass groups information to SonarQube